go-controller/pkg/node/node.go

```diff
@@ -244,11 +245,10 @@ func (n *OvnNode) Start() error {
 		if err != nil {
 			return err
 		}
+		wg.Add(1)
 		go func() {
-			err := nodeController.Run(n.stopChan)
-			if err != nil {
-				klog.Error(err)
-			}
```

Comment:
any reason you dropped the error logging?

Reply:
@danwinship yeah, after Dave Tucker's workqueue changes that function always returns `nil` so the return value was pointelss.

Reply:
oh, maybe get rid of the return value too then

Reply:
Yeah, I did.